#!/usr/bin/env python
PACKAGE = "fsm"

import roslib;roslib.load_manifest(PACKAGE)
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("isToWaypoint", bool_t, 0, "Switch to waypoint following", False)

exit(gen.generate(PACKAGE, "fsm", "fsm_monitor"))
